paper-input.html

```diff
@@ -90,6 +90,14 @@
       },
 
       /**
+       * Marks the `input` as required.
```

Comment:
How about "set to true to mark the `input` as required" to match the other doc string?